src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -40,6 +40,7 @@ class IntroductionPage extends React.Component {
           pathname={this.props.location.pathname}
           user={user}
           authenticate={this.authenticate}
+          noPrefill
```

Comment:
Is `skipPrefill` maybe a better name? Feels like it's a little more descriptive of what's going on.